PC 31823/fix public api null remaining quantity (#14403)
pass-culture/pass-culture-main: api/documentation/docs/understanding-our-api/managing-bookings/connection-with-ticketing-system.md

| Key              | Type | Nullable | Explanation |
| :---------------- | :------ | :----: | :-------- |

> | **remainingQuantity** | Integer | **`true`** | Number of tickets still available after this booking (for us to update our stock). Please note that an unlimited quantity means `null` not "unlimited" or anything else and that this field is **mandatory**. |

Reviewer: Question : est-ce que ça serait pas plus clair formulé ainsi ?
```suggestion
| **remainingQuantity** | Integer | **`true`** | Number of tickets still available after this booking (for us to update our stock). To indicate an unlimited quantity, set `remainingQuantity` to `null` (not to `"unlimited"`). Please note that this field is **mandatory**. |
```
